mivs/model_checks.py

```diff
@@ -14,7 +14,7 @@ def _is_invalid_url(url):
 
 @validation.IndieStudio
 def valid_url(studio):
```

Comment:
I added some code which automatically prepends ``http://`` if the URL isn't already fully qualified.